apps/charterafrica/src/lib/ecosystem/airtable/processData.js

```diff
@@ -0,0 +1,194 @@
+import * as Sentry from "@sentry/nextjs";
+
+function getter(data, key) {
+  return key ? data?.[key] : null;
+}
+
+function mapSupportersToFields(supporters, config, tableData) {
```

Comment:
```suggestion
function mapSupporterIdsToObjects(supporterIds, config, { partnersData }) {
```

Since it's raw data from Airtable (i.e. contains fields, etc.), I think we should call it `partnersData` rather than just partners to emphasise it hasn't been processed.